10-stretch-slim/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Debian-based image maintained by Node, see https://hub.docker.com/_/node/ for a full list of available images.
+# Source for this base image can be found at https://github.com/nodejs/docker-node/tree/master/10/stretch-slim
+FROM node:10.5.1-stretch-slim
```

Comment:
In the node-8 version I kept this to the major version only. It was the same with the Alpine images. So I might suggest we change this to node:10-stretch-slim?